lib/init-input.js

```diff
@@ -1,3 +1,5 @@
+/* jshint ignore: start */
```

Comment:
Huh. Weird. Carry on then.

If you could put that explanation in a comment at the top then that would be helpful.